arch/xtensa/src/esp32/esp32_spiflash.c

```diff
@@ -2660,4 +2750,200 @@ bool esp32_flash_encryption_enabled(void)
   return enabled;
 }
 
+#ifdef CONFIG_ESP32_SPI_FLASH_MMAP
+
+/****************************************************************************
+ * Name: esp32_spiflash_mmu_init
+ *
+ * Description:
+ *   Initialize MMU for accessing SPI flash by I-Bus.
+ *
+ * Input Parameters:
+ *   None
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+void esp32_spiflash_mmu_init(void)
+{
+  uint32_t regval;
+
+  /* Initialize I-Bus 2 section, initial value can't be invalid */
+
+  for (int i = IBUS2_PAGE_START; i < IBUS2_PAGE_END; i++)
```

Comment:
`int i;` to the beginning of function

Reply:
I would say that it is not mandatory for arch specific code